### PR TITLE
Fix string literal

### PR DESCRIPTION
**Pull request by yoshihiro503**
Support multibyte strings and add test cases.

### CODE REVIEW

File: test/test_string.erl [hunk 1]
```diff
@@ -0,0 +1,9 @@
+-module(test_string).
+
+-export([f/0, g/0]).
+
+%% test case for string literal (ASCII)
+f() -> "abc%/_hoge".
+
+%% test case for string literal (japanese)
+g() -> "いろはにほへとちるぬるを".
```
Comment (amutake) [inline test/test_string.erl]: Sorry, could you add a test case for empty string (`""`)?

Comment (yoshihiro503) [reply test/test_string.erl]: yes.

Comment (amutake) [reply test/test_string.erl]: Thank you!
